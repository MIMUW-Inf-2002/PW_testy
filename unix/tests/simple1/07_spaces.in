run ./04_kill





out 0

err 0
    


run ./04_kill


       

