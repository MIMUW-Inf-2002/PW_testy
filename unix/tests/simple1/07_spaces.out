Task 0 started: pid 55136.
Task 0 stdout: ''.
Task 0 stderr: ''.
Task 1 started: pid 55138.
======
Task 0 ended: signalled.
Task 1 ended: signalled.
======
